.gitignore

```diff
@@ -1,3 +1,4 @@
-# Custom ignore rules:
-## Python general
```

Comment:
You should leave at least one header comment explaining the rules below.
See how it's done here: https://github.com/github/gitignore/blob/master/Python.gitignore